shell/components/Tabbed/index.vue

```diff
@@ -66,6 +66,11 @@ export default {
     resource: {
       type:    Object,
       default: () => {}
+    },
+
+    showExtensionTabs: {
```

Comment:
I found an issue where when extensions added tabs (kubewarden and the compliance tab) they would be added to all the tables on the page included in the resource detail drawer. It was breaking one of the harvester drawers.